src/cell_group_factory.cpp

```diff
@@ -31,6 +32,9 @@ cell_group_ptr cell_group_factory(const recipe& rec, const group_description& gr
     case cell_kind::data_spike_source:
         return make_cell_group<dss_cell_group>(group.gids, rec);
 
+    case cell_kind::inhomogeneous_poisson_spike_source:
```

Comment:
We should put an issue up about making this generic so that cell groups can be loaded dynamically rather than requiring someone to find this case to add new groups.